augur/utils.py

```diff
@@ -544,21 +546,30 @@ def run_shell_command(cmd, raise_errors = False, extra_env = None):
     overlayed onto the default subprocess environment.
     """
     env = os.environ.copy()
+    out = ''
 
     if extra_env:
         env.update(extra_env)
 
     try:
+        shargs = ['-c', "set -euo pipefail; " + cmd]
+
+        if os.name == 'posix':
+            shellexec = ['/bin/bash']
+        else:
+            # We try best effort on other systems. For now that means nt/java.
+            shellexec = ['env', 'bash']
+
         # Use check_call() instead of run() since the latter was added only in Python 3.5.
-        subprocess.check_call(
-            "set -euo pipefail; " + cmd,
-            shell = True,
-            executable = "/bin/bash",
-            env = env)
+        out = subprocess.check_output(
+            shellexec + shargs,
+            shell = False,
+            stderr=subprocess.STDOUT)
```

Comment:
This change from `check_call` to `check_output` drops the environment modification `env = env`. That's an important part that should stay.

Reply:
Oops!